p2p/peer.py

```diff
@@ -347,7 +350,7 @@ def close(self):
                 self.logger.debug("%s disconnected: %s", self, e)
                 return
 
-    async def read_msg(self) -> Tuple[protocol.Command, protocol._DecodedMsgType]:
+    async def read_msg(self) -> Tuple[protocol.Command, Dict[str, Any]]:
```

Comment:
Why this change?  This method can return things other than a dict, that's why it used `protocol._DecodedMsgType`

Reply:
True, some temporary change that made it through my own review.

Reply:
Are you going to revert it?

Reply:
Sorry, fixed by https://github.com/ethereum/py-evm/pull/947/commits/1bbab42b47e15f6453894dcc993e1d9bab925a05